OracleGoldenGate/21c/README.md

```diff
@@ -141,6 +143,15 @@ Oracle GoldenGate Administration Client for Oracle
 Version 21.3.0.0.0 ...
 ```
 
+### Running scripts before setup and on startup
+The container images can be configured to run scripts before setup and on startup. Currently, sh extensions are supported. For setup scripts just mount the volume /u01/ogg/scripts/setup or extend the image to include scripts in this directory. For startup scripts just mount the volume /u01/ogg/scripts/startup or extend the image to include scripts in this directory. Both of those locations are static and the content is controlled by the volume mount.
```

Comment:
Please back-tick quote literals like `.sh` and directory names

Reply:
added Backtick quote